### ISSUE TITLE
Serial redirect to file

### ISSUE THREAD

**Issue by ardenee** [fbelavenuto/arpl#541]
Hi, 

Just wondering if it is possible to redirect the serial output to a file for debugging, currently when DSM boots and the system freezes, I've no idea as to why, this may help with such crashing going forward.

While I am hoping to get a serial adapter to hook up to the system, a file output option would be handy for quick debugging too.

Also, I see loglevel is defaulted to 15, can this be increased or is that the max?

**Comment by ardenee** [comment]
So, I managed to get the serial hooked up and it looks like it may be the CPU 'count' that’s causing the crash? A lot of forums point to resolving this by adding additional CPU’s, from 1 to 2. They are usually talking about in virtualization configurations though, but does the loader in a way do this too, is it limited to just 1 CPU?

Another possible cause might be a 'broken Initrd image'.

[serial-crash-cpu.txt](https://github.com/fbelavenuto/arpl/files/10521306/serial-crash-cpu.txt)

**Comment by fbelavenuto** [comment]
Please try to add the variable "disable_mtrr_trim" with no value in cmdline menu:

https://github.com/fbelavenuto/arpl/wiki/Add-an-cmdline-synoinfo-entry

Boot and test

Possibly related with #537

**Comment by ardenee** [comment]
Thanks fbelavenuto! that did the trick.

Ended up getting stuck at the no nic issue, but it got past the kernel panic. 

I ended up using beta 3 and was able to get the nic working and log into the old DSM (MFA reset needed tho) but it's up and running now.

Only issue now is all the drives are not showing up for some reason, only 7 of them instead of 13 but I'll look into this tomorrow. 

Thanks again!
